### PR TITLE
update Site editor tracks e2e tests for new theme, gutenberg changes, and edge environment support.

### PR DESCRIPTION
**Pull request by Addison-Stavlo**
#### Changes proposed in this Pull Request

Updates the site editor tracks e2e tests for proper GUTENBERG_EDGE environment support, as well as required updates for recent changes in Gutenberg and the theme for the test site.

#### Testing instructions

* Run this test suite locally and verify all tests path with the GUTENBERG_EDGE env variable in both `true` and `false` states.  ex - `env GUTENBERG_EDGE=true ./node_modules/.bin/mocha specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js`

Related to #

### CODE REVIEW

**Comment by jeyip** [comment]
## Testing
### Requirements
- [x] When the `GUTENBERG_EDGE` flag is true, the `wp-calypso-gutenberg-site-editor-tracking-spec.js` e2e test is run in an edge site environment
- [x] When the `GUTENBERG_EDGE` flag is false, the `wp-calypso-gutenberg-site-editor-tracking-spec.js` e2e test is run in a non-edge site environment
- [x] `wp-calypso-gutenberg-site-editor-tracking-spec.js` test suite passes with a true and false `GUTENBERG_EDGE` flag

**Review by jeyip — APPROVED** [review]

File: test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
+/**
+ * External dependencies
+ */
```
Comment (creativecoder) [inline test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js:3]: FYI, it appears we don't need these import comments any longer. More info: p4TIVU-9O9-p2

Comment (Addison-Stavlo) [reply test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js:3]: 🤔   The linter forced me to do this after I rebased on Tuesday, there was no issue before that.

Comment (creativecoder) [reply test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js:3]: 😕

cc @scinos

Comment (scinos) [reply test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js:3]: I can delete those comments in `trunk` and eslint doesn't seem to complain to me.

Also all code style checks for this branch have been always green (But maybe some builds are missing?): 
image

Maybe it was a polluted `node_modules` that still had the old rule active or something like that.

Comment (Addison-Stavlo) [reply test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js:3]: > Maybe it was a polluted node_modules that still had the old rule active or something like that.

Seems to be!  After cleaning and reinstalling the node modules the linter isn't mad at me in these files.  Odd the linter never said anything until Tuesday though?  Oh well. 😆

**Pull request merged**